data/json/recipes/other/tools.json

```
@@ -2752,5 +2752,25 @@
     "autolearn": true,
     "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "rock", 1 ], [ "sharp_rock", 1 ] ], [ [ "stick", 1 ], [ "2x4", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "pocketwatch_crude",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "skills_required": [ "mechanics", 4 ],
+    "time": "2 h",
+    "autolearn": [ [ "mechanics", 3 ] ],
+    "book_learn": [ [ "textbook_mechanics", 4 ], [ "manual_mechanics", 4 ], [ "textbook_fabrication", 4 ] ],
```

Forgot to add this suggestion hecc
```suggestion
    "difficulty": 5,
    "skills_required": [ "mechanics", 3 ],
    "time": "2 h",
    "autolearn": [ [ "mechanics", 3 ] ],
    "book_learn": [ [ "textbook_mechanics", 3 ], [ "manual_mechanics", 3 ], [ "textbook_fabrication", 3 ] ],
```